DirectXTex/BCDirectCompute.cpp

```diff
@@ -371,7 +371,7 @@ HRESULT GPUCompressBC::Prepare(size_t width, size_t height, uint32_t flags, DXGI
 
 
 //-------------------------------------------------------------------------------------
-HRESULT GPUCompressBC::Compress(const Image& srcImage, const Image& destImage)
+HRESULT GPUCompressBC::Compress(const Image& srcImage, const Image& destImage, ProgressProc progressProc)
```

Comment:
Have you seen GPU compression to be slow enough to need a callback?

Reply:
No, I have not. IIRC, I added that callback for parity with the CPU-based compressor code. 
The callback is only used by `HRESULT DirectX::CompressEx(ID3D11Device* pDevice,
    const Image& srcImage,
    DXGI_FORMAT format,
    TEX_COMPRESS_FLAGS compress,
    float alphaWeight,
    ScratchImage& image,
    ProgressProc progressProc) noexcept`, I could remove it and change that method to report 0% and 100%.

Reply:
I removed the progress reporting code from the DirectComput inner loop.